libs/templates/src/lib/home/sections/account-section/data.tsx

```diff
@@ -8,7 +8,7 @@ export const TableData: TableContent = {
   description:
     'Direct market access with fast order execution and no dealing desk intervention.',
   cta: (
-    <Button variant="secondary" className="w-1/2">
+    <Button variant="secondary" colorStyle="black" size="lg" className="w-1/2">
```

Comment:
I think the we need to adjust the props in the quill-design. 

```suggestion
    <Button variant="secondary" color="black" size="lg" className="w-1/2">
```

Reply:
we can't do it, color is a html attribute man. so we can't use it.